app/src/main/java/net/frju/flym/ui/entrydetails/EntryDetailsView.kt

```diff
@@ -76,6 +61,32 @@ class EntryDetailsView @JvmOverloads constructor(context: Context, attrs: Attrib
 
 
     init {
+        //Change theme if light theme is used
+        BACKGROUND_COLOR = if (PrefUtils.getBoolean(PrefUtils.DARK_THEME, true)) "#202020" else "#f6f6f6"
```

Comment:
instead of putting some variables outside the class and some inside, it's better to put all of them inside the class and use the context to ask for theme attributes

Something like:
```
 class EntryDetailsView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : WebView(context, attrs, defStyleAttr) {
    private val BACKGROUND_COLOR = context.colorAttr(R.attr.colorBackground)
```

It is possible to add custom attributes for the other colors and define them in the dark and light theme